ext/spl/spl_array.c

```diff
@@ -1883,6 +1883,24 @@ SPL_METHOD(Array, __unserialize)
 }
 /* }}} */
 
+/* {{{ proto void Array::__debugInfo() */
+SPL_METHOD(Array, __debugInfo)
+{
+	HashTable *ht;
+	int is_temp;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	ht = spl_array_get_debug_info(getThis(), &is_temp);
```

Comment:
As this is the only place where this function is used, I'd suggest moving the is_temp logic into it and dropping the argument.

Reply:
Ah, nice!